ui/components/app/asset-list/asset-list.buy-receive.test.js

```diff
@@ -17,7 +17,7 @@ const render = (
     ...mockState,
     metamask: {
       ...mockState.metamask,
-      providerConfig: { chainId },
+      providerConfig: { type: 'test', chainId },
       cachedBalances: {
         [CHAIN_IDS.MAINNET]: {
           [selectedAddress]: balance,
```

Comment:
Same here. I don't understand why we would need to update this file?

Reply:
See above

Reply:
Resolved for same reason as above